src/archive/mod.rs

```diff
@@ -366,7 +398,13 @@ impl<'a> Archive<'a> {
             let name = member.raw_name();
             if name == INDEX_NAME {
                 let data: &[u8] = buffer.pread_with(member.offset as usize, member.size())?;
-                index = Index::parse_sysv_index(data)?;
+                index = if sysv_symbol_index_seen {
```

Comment:
Maybe we should return an error if this is the third time or more.

Reply:
Or ignore them.

Reply:
I'm not sure about this. We'll have to track then number of occurrences of index, including BSD index to have the behaviour consistent and allow only one index to be present (exept if it's Windows linker member). I think it would be better to tackle it systematically in a separate pull-request.

Reply:
The current behaviour would be to threat all the subsequent SysV symbol indexes as Windows linker members, similar to running into multiple BSD symbol indexes.

Reply:
I'm fine with doing this in another PR, but I don't like treating subsequent SysV symbol index as Windows linker members because that's the sort of sloppy parsing that caused the errors this PR is fixing. It's better to not support something, rather than try to parse it incorrectly.

Reply:
I agree that the parsing should be too strict. It's just the question of how much complexity is it worth introducing for. I've drafted a separate PR #175 to figure the best way forward.